packages/react-native-editor/__device-tests__/helpers/caps.js

```diff
@@ -1,5 +1,11 @@
+/**
+ * Internal dependencies
+ */
+import deviceConfig from './device-config.json';
+
+const { ios: iOSConfig, android: androidConfig } = deviceConfig;
```

Comment:
Is there a reason to avoid de-structuring the import? Or is this invalid syntax?
```suggestion
import {
	ios as iOSConfig,
	android as androidConfig,
} from './device-config.json';
```

Reply:
>Is there a reason to avoid de-structuring the import?

Not at all, initially I was using `deviceConfig` directly 😄 Thanks for the suggestion!